src/convertToCase/inputValidation.js

```diff
@@ -0,0 +1,33 @@
+const allCases = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+function inputValidation(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message:
+        // eslint-disable-next-line max-len
+        'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!targetCase) {
+    errors.push({
+      message:
+        // eslint-disable-next-line max-len
+        '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  } else if (!allCases.includes(targetCase.toUpperCase())) {
```

Comment:
It's a good practice to use constants for hardcoded strings that are used in multiple places. You can create a constant for the error message about unsupported cases and use it here.